Vagrantfile

```diff
@@ -67,7 +66,9 @@ Vagrant.configure(2) do |config|
   config.vm.provision "shell", path: "bootstrap.sh",
     env: {"AWS_ACCESS_KEY_ID": ENV["AWS_ACCESS_KEY_ID"],
           "AWS_ACCESS_SECRET": ENV["AWS_ACCESS_SECRET"],
-          "AWS_REGION": ENV["AWS_REGION"]}
+          "AWS_REGION": ENV["AWS_REGION"],
+          "SQS_IDENT": ENV["SQS_IDENT"]},
+    run: "always"
```

Comment:
If we run `bootstrap.sh` every time, does that mean the database gets wiped and the S3 bucket gets emptied every time we `vagrant up`, even after a `vagrant suspend`? That seems... not great.

Reply:
It's not great. I'll add some logic in the bash script that asks you if you want to wipe stateful data.